rotkehlchen/chain/ethereum/modules/airdrops/decoder.py

```diff
@@ -107,14 +135,17 @@ def _decode_fox_claim(self, context: DecoderContext) -> DecodingOutput:
             return DEFAULT_DECODING_OUTPUT
 
         raw_amount = hex_or_bytes_to_int(context.tx_log.data[64:96])
-        amount = asset_normalized_value(amount=raw_amount, asset=self.fox)
+        amount = token_normalized_value_decimals(
+            token_amount=raw_amount,
+            token_decimals=DEFAULT_TOKEN_DECIMALS,
```

Comment:
Are they all at 18 decimals. Can you write small comment here and in all others here like this:
```suggestion
            token_decimals=DEFAULT_TOKEN_DECIMALS,  # fox 18 decimals
```

Reply:
Yes! I've checked all of them before using the constant